.github/workflows/cs.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: '8.1'
```

Comment:
Might this point out to use PHP language features not available in `7.4`?
Our is our ruleset properly configured for this?

Reply:
No, CS is just that, CS AFAIK it doesn't do any language feature pointing out unfortunately.

Reply:
Let's expore that in a different PR, this was mostly done so you can use Laravel 9 (latest) in local development :)